### ISSUE TITLE
finetune strategy on 1k data

### ISSUE THREAD

**Issue by pawopawo** [Alibaba-MIIL/ImageNet21K#21]
After using single label to pretrain on 21k data, what is the finetune strategy on 1k data? E.g. MobilenetV3_large_100

**Comment by mrT23** [comment]
why do you want to finetune on 1k ? i dont really see the point
anyway, similar to standard 1k training, only with lower learning rate, and you can try adam instead SGD.

**Comment by pawopawo** [comment]
![image](https://user-images.githubusercontent.com/38682321/123894197-4e78f500-d990-11eb-9a48-00fbdf65ad75.png)
Thanks. I want to reproduce the results in the table.
The finetune result on Imagenet-1K is only with lower learning rate and adam?

**Comment by mrT23** [comment]
basically yes (compared to standard 21K training)

**Comment by cissoidx** [comment]
@pawopawo have you reproduced the acc?
@mrT23 In the paper, Table 3, the imagenet21k pretrained model were obtained using single label or semantic softmax training strategy? Did the official site mention if they actually use which strategy? Or by default people are all using semantic softmax way?
